### DISCUSSION TITLE
多个路由的情况下只有一个能访问

### DISCUSSION THREAD

**Discussion by hyunrealshadow** [alibaba/higress#448] [Q&A]
我有一个泛域名证书，通过console更新到了控制台中，根据不同服务分配了不同的子域名
但是这些子域名中只会有一个能访问，其他都会返回404
目前用的是一个k3s单节点集群
请问以下配置哪里可能有问题？
![higress_1](https://github.com/alibaba/higress/assets/36305150/9d3bb1aa-800e-4f9f-9ef5-f12439484667)
![1690179397014](https://github.com/alibaba/higress/assets/36305150/41c0c851-2a27-4709-a15b-4dc1efa5ca33)
![image](https://github.com/alibaba/higress/assets/36305150/ab8fff4a-3188-4f80-915d-d9b994f5828f)

**Comment by johnlanni** [comment]
已经在钉钉群沟通了哈，是开启HTTP2下会有这个影响，主要是目前域名和证书映射到 Gateway 的模型导致的，我们后面可以优化下。目前先关闭下 ALPN H2
